ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -99,10 +100,39 @@ public void stop(final Promise<Void> stopPromise) {
     }
   }
 
+  // Creates a new server, rather than simply stopping and restarting, in order to reload TLS certs
+  public void restartServer() {
```

Comment:
As previous comment, seems overcomplex - just stopped and starting the Server should do the trick?

Reply:
As above. I'll update this comment to be more explicit for the reasoning.